person.rb

```diff
@@ -0,0 +1,28 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    super(name)
```

Comment:
- [ ] The super( ) keyword doesn't need arguments please 🙏🏽 remove it otherwise your app will have the following issue 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/225926941-b71d52fa-c94c-4558-a398-5c566c6a5a5d.png)

If you'd like to review more about the super keyword please read this [article](https://medium.com/rubycademy/the-super-keyword-a75b67f46f05)